src/main/kotlin/blackjack/model/Dealer.kt

```diff
@@ -0,0 +1,27 @@
+package blackjack.model
+
+import blackjack.controller.Deck
+
+data class Dealer(override val name: String = "Dealer") : Playable() {
+    fun shouldDrawCardOrNot(): Boolean {
+        val score = calculateHand()
+        return score <= 16
+    }
+
```

Comment:
What do you think is the role of the View in the MVC pattern?
Could you explain how you see the responsibilities being divided between the Model and the View?
I think that would help me better understand your intent. 💭